scripts/workflow-state.sh

```
@@ -534,6 +670,129 @@ cmd_next_action() {
+                if [ "$integration_status" = "passed" ]; then
+                    echo "AUTO:refactor-review"
+                elif [ "$integration_status" = "failed" ]; then
+                    echo "AUTO:delegate:--fixes"
+                else
+                    echo "WAIT:in-progress:integrating"
+                fi
+                ;;
+            implement)
+                # Polish track only - check if implementation is done
+                # For polish, we check if goals are being verified
+                local tests_pass=$(jq -r '.validation.testsPass // false' "$state_file")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)